src/components/VoiceCompair/VoiceCompair.js

```diff
@@ -38,6 +38,18 @@ const VoiceCompair = props => {
   const [tamilRecordedAudio, setTamilRecordedAudio] = useState('');
   const [tamilRecordedText, setTamilRecordedText] = useState('');
 
+  const [asr_language_code, set_asr_language_code] = useState('ai4bharat/whisper-medium-en--gpu--t4');
+
+  useEffect(() => {
+	if (lang_code === 'hi')
+	{
+		set_asr_language_code('ai4bharat/conformer-hi-gpu--t4');
+	}
+	else
+	{
+		set_asr_language_code('ai4bharat/whisper-medium-en--gpu--t4');
```

Comment:
can we set the asr language code in ASR_REST_URLS

Reply:
@gouravmore In that case we have to add whole URL of the API (host,endpoint & language service id). Also, we don't know the language service id of other languages other than tamil, hindi, english & bengali. So, it's better to ASR_REST_URLS should have the domain & language service id will be get appended depend on the active language.

Reply:
Ok, but instead of hardcoding it in the switch condition, can we define this as a constant at the top of the variable declaration section? This way, we can easily manage and update the URLs for different languages without modifying the switch condition

Reply:
Fixed